packages/atomic/scripts/set-headless-private.js

```diff
@@ -0,0 +1,19 @@
+const util = require('util');
+const fs = require('fs');
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
+
+async function setHeadlessPrivate() {
+  const headlessPackageJsonPath = 'headless/package.json';
+  const headlessPackageJSONData = await readFile(headlessPackageJsonPath);
+  const headlessPackageData = JSON.parse(headlessPackageJSONData);
+
+  headlessPackageData.private = true;
```

Comment:
mostly a technicality, to be clear its not meant to be published

Reply:
Is this still needed if we specify the packages to publish [here](https://github.com/coveo/ui-kit/pull/1500/files#r753361851)?

Reply:
Just make sure this step/script is still needed, then 🚢

Reply:
It's not really needed, kept it for insurance as we will never publish it by mistake.

I'll remove it